app/views/idv/doc_auth/_back.html.erb

```diff
@@ -21,11 +21,10 @@ classes << local_assigns[:class] if local_assigns[:class]
   <div class="margin-top-5 padding-top-2 border-top border-primary-light">
     <% if local_assigns[:action] %>
       <%= button_to(
-        text,
         path,
         method: :put,
         class: [*classes, 'usa-button usa-button--unstyled']
-      ) %>
+      ) { '‹ ' + t('forms.buttons.back') } %>
```

Comment:
Should this be....

```suggestion
      ) { text } %>
```

?

Same result either way I suppose. Alternatively, for consistency we could update the `else` branch of this condition to inline the text as well, and remove the `text` variable assignment.